client/src/components/molecules/DonationItemForm.tsx

```diff
@@ -0,0 +1,307 @@
+                            onDropdownOpen={() => {
+                                setIsConditionDropdownOpen(true);
+                            }}
+                            isDropdownOpened={isConditionDropdownOpen}
+                        />
+                    }
+                />
+                <FormItem
+                    className="item-age-field"
+                    formItemName="Age of Item *"
+                    errorString=""
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    tooltipText="Please select the approximate number of years you have had this item"
+                    inputComponent={
+                        <ScrollableDropdown
+                            trigger={
+                                <TextField
+                                    input={ItemAgeToDescriptionMap.get(age) ?? ""}
+                                    isDisabled={false}
+                                    isErroneous={false}
+                                    onChange={() => {}}
+                                    name="age"
+                                    placeholder=""
+                                    type="text"
+                                    showRedErrorText={true}
+                                    readOnly={true}
+                                    iconClassName="bi bi-caret-down-fill"
+                                />
+                            }
+                            dropdownItems={Array.from(ItemAgeToDescriptionMap.entries()).map(
+                                ([ageKey, description]) =>
+                                    ageKey > 0 && (
+                                        <span key={ageKey} onClick={() => onAgeChange(ageKey)}>
+                                            {description}
+                                        </span>
+                                    )
+                            )}
+                            onDropdownClose={() => {
+                                setIsAgeDropdownOpen(false);
+                            }}
+                            onDropdownOpen={() => {
+                                setIsAgeDropdownOpen(true);
+                            }}
+                            isDropdownOpened={isAgeDropdownOpen}
+                        />
+                    }
+                />
+                <FormItem
+                    className="item-quantity-field"
+                    formItemName="Quantity *"
+                    errorString=""
+                    isDisabled={false}
+                    showErrorIcon={false}
+                    tooltipText="Please input the number of identical items you are requesting to donate"
+                    inputComponent={
+                        <TextField
+                            input={quantity.toString()}
+                            isDisabled={false}
+                            isErroneous={isQuantityErrorneous}
```

Comment:
When I tested this out, and left the placeholder value of 1 and saved item, no errors occurred. Idk if this matters (personally I dont care about this) but design is that placeholder is not an inputted value

Reply:
My impression was that the default quantity is 1. But none of the specs mention it
I'll ask Amanda again... maybe she mentioned it verbally